tests/integration-tests/configs/common/not_released_osses.yaml

```diff
@@ -15,15 +15,15 @@ cloudwatch_logging:
       # 2) run the test for all OSes with slurm
       - regions: ["ap-east-1"]
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
-        oss: ["ubuntu2204"]
```

Comment:
Nit: Something we should review later, but we shouldn't have tests that run on different oses in the config.  Given that this is specifically, not_released_oses, it should only test on the one OS